src/clover/fileparser.py

```diff
@@ -2206,6 +2210,7 @@ def parse_input_files(  # pylint: disable=too-many-locals, too-many-statements
             - finance_inputs,
```

Comment:
## 💚 Mypy note
You'll need to add into the above type annotation that you're now returning the grid attributes as well.